README.md

```diff
@@ -76,3 +76,15 @@ datasources:
       accessKey: '<your access key>'
       secretKey: '<your secret key>'
 ```
+
+## How to build this datasource
+You will need to install [Node.js](https://nodejs.org/en/), [Yarn](https://yarnpkg.com/), [Go](https://golang.org/), and [Mage](https://magefile.org/) first.
+1. `yarn install --frozen-lockfile`
+1. `yarn dev` — will build the frontend changes
+1. `mage build:backend` — will build the backend changes
+1. (Optional) `mage -v buildAll` — this is optional if you need backend plugins for other platforms
+1. [Sign the plugin](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/), or configure Grafana to [load the unsigned plugin](https://grafana.com/docs/grafana/latest/plugins/plugin-signatures/#allow-unsigned-plugins).
```

Comment:
This one I'm not sure about because 3rd parties can not actually sign...  Perhaps we just say to run grafana in developer mode?

Reply:
Can they sign a plugin for a particular hostname?

Reply:
not really -- the signature is the thing that says it is official.  If it is for development, just run grafana in dev mode and all is well.  

If they want their own version of the plugin, they need to change the id and then can sign it with a personal key (with hostname etc), but we don't let 3rd parties sign plugins they do not "own"

Reply:
Let me fix this in another PR then.